test/libsolidity/lsp/references/import_aliases_2.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import {RGBColor as A, RGBColor as B, RGBColor as C} from "../goto/lib.sol";
```

Comment:
Can you add a check pointing at `RGBColor` to see if it will find its aliases or not? Best if you could also add something else called `RGBColor` somewhere in the file.